docs/content/en/docs/Tutorials/add-username-password-authentification-k3os-ssh/index.md

```diff
@@ -5,6 +5,8 @@ description: >
   This article explains how to enable the classic username / password authentification for SSH in k3os 
```

Comment:
Why would you want to add SSH password authentification: what actually changes?
And what are the dangers?